docs/eloquent-models/relationships.txt

```diff
@@ -0,0 +1,514 @@
+   :dedent:
+
+The following sample code shows how to instantiate a model for each class
+and add the relationship between them. Click the :guilabel:`View Output` 
+button to see the data created by running the code:
+
+.. io-code-block::
+
+   .. input:: /includes/eloquent-models/relationships/RelationshipController.php
+      :language: php
+      :dedent:
+      :start-after: begin many-to-many save
+      :end-before: end many-to-many save
+
+   .. output::
+      :language: json
+      :visible: false
+
+      // Documents in the "planets" collection
+      [
+        {
+          _id: ObjectId('65e1043a5265269a03078ad0'),
+          name: 'Earth',
+          // ...
+          space_explorer_ids: [
+            '65e1043b5265269a03078ad3',
+            '65e1043b5265269a03078ad4',
+            '65e1043b5265269a03078ad5'
+          ],
+        },
+        {
+          _id: ObjectId('65e1043a5265269a03078ad1'),
+          name: 'Mars',
+          // ...
+          space_explorer_ids: [ '65e1043b5265269a03078ad4', '65e1043b5265269a03078ad5' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad2'),
+          name: 'Jupiter',
+          // ...
+          space_explorer_ids: [ '65e1043b5265269a03078ad3', '65e1043b5265269a03078ad5' ]
+        }
+      ]
+
+      // Documents in the "space_explorers" collection
+
+      [
+        {
+          _id: ObjectId('65e1043b5265269a03078ad3'),
+          name: 'Tanya Kirbuk',
+          // ...
+          planet_ids: [ '65e1043a5265269a03078ad0', '65e1043b5265269a03078ad2' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad4'),
+          name: 'Mark Watney',
+          // ...
+          planet_ids: [ '65e1043a5265269a03078ad0', '65e1043a5265269a03078ad1' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad5'),
+          name: 'Jean-Luc Picard',
+          // ...
+          planet_ids: [
+            '65e1043a5265269a03078ad0',
+            '65e1043a5265269a03078ad1',
+            '65e1043b5265269a03078ad2'
+          ]
+        }
+      ]
+
+The following sample code shows how to access the related models by using
+the dynamic properties as defined in the example classes.
+
+.. literalinclude:: /includes/eloquent-models/relationships/RelationshipController.php
+   :language: php
+   :dedent:
+   :start-after: begin many-to-many dynamic property example
+   :end-before: end many-to-many dynamic property example
+
+
+.. _laravel-embedded-document-pattern:
+
+Embedded Document Pattern
+-------------------------
+
+In MongoDB, the embedded document pattern adds the related model's data into
+the parent model instead of keeping foreign key references. This pattern
+when you must optimize for one or more of the following requirements:
+
+- Keep associated data together in a single collection
+- Perform atomic updates on multiple fields of the document and the associated
+  data
+- Reduce the number of reads required to fetch the data
```

Comment:
S: based on the way the introduction to this list ends, I think you should change the format of these items
```suggestion
- You must keep associated data together in a single collection
- You must perform atomic updates on multiple fields of the document and the associated
  data
- You must reduce the number of reads required to fetch the data
```

Reply:
Good idea. I decided to convert the verb to a noun by using the gerund.
I think this agrees better with the list introduction.